turk-one/app/Views-Controllers/Admin/admin-datatable.js

```diff
@@ -0,0 +1,228 @@
+if (Meteor.isClient) {
+
+	Meteor.subscribe("workerTickets");
+
+	// worker review page
+	function createMongoSelectorFromConditionals(arrayOfConditionals) {
+		if (arrayOfConditionals.length > 0) {
+			return {$or: arrayOfConditionals};
+		} else {
+			return {}
+		}
+	}
+
+	Template.adminDataTable.helpers({
+		workerTickets: function () {
+			// TODO: Store session keys someplace so we don't have to remember the keys specifically each time
```

Comment:
Yes! Session keys not being strings is something I would **love**. In Swift, I would implement these as enums to make them less stringy and more enforced... Wonder what standard practice is for doing this in Javascript.